lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -134,9 +134,13 @@
     prefix: aws_region
 # Set individual variables with compose
 compose:
-  # Use the private IP address to connect to the host
+  # Use the private IP address to connect to the host and add additional variables as hostvars
+  # EC2 attribute private_ip_address will be mapped to hostvars ansible_host and internal_ip_adddress
+  # EC2 attribute public_ip_address will be mapped to hostvars public_ip_address
   # (note: this does not modify inventory_hostname, which is set via I(hostnames))
   ansible_host: private_ip_address
+  public_ip_address: public_ip_address
```

Comment:
I don't understand why this is useful. public_ip_address already equals public_ip_address so this does nothing.